README.md

```diff
@@ -69,8 +69,12 @@ export PYTHONPATH=$(pwd)
 ```
 
 ### Run tests
+
+To run all tests, just run
+
 ```
 docker-compose up
 ```
 
+While the `docker-compose` stack is up, you can also run the tests from the CLI via `pytest tests/ --integration --local` 
 
```

Comment:
That's actually pretty annoying because the esque docker will still be built. Should we provide a possibility to actually run the stack without esque?

Reply:
Thought about that too, but i'm not sure how i'd compose the compose files (hehe) to make this work with both development options without making them more complicated

Reply:
We could create a base `docker-compose.yml` with the stack. And add the integration test in a second one inheriting from the first: https://docs.docker.com/compose/extends/